src/ansible_navigator/actions/builder.py

```diff
@@ -0,0 +1,102 @@
+"""Builder subcommand implementation.
+
+Importing this module registers this subcommand in the external
+global subcommand registry.
+"""
+import os
+import shutil
+import sys
+
+from typing import Optional
+from typing import Tuple
+
+from ..app import App
+from ..configuration_subsystem import ApplicationConfiguration
+from ..configuration_subsystem.definitions import Constants
+from ..runner import Command
+from . import _actions as actions
+
+
+@actions.register
+class Action(App):
+    """Run the builder subcommand."""
+
+    KEGEX = "^b(?:uilder)?$"
```

Comment:
since this is stdout only, we can probably have this be just "builder", which will leave `:b` for something in the TUI... (unless of course we think we'll add it someday)

Reply:
Come to think of it, it's not worth changing until we need b